function.php

```diff
@@ -0,0 +1,107 @@
+        <li><a class=\"produce\" href=\"index.php?category=produce\">Produce</a></li>
+        <li><a class=\"other\" href=\"index.php?category=other\">Other</a></li>");
+        break;
+        case 'pantry':
+            return ("<li><a class=\"all\" href=\"index.php?category=all\">All</a></li>
+        <li><a class=\"pantry current\" href=\"index.php?category=pantry\">Pantry</a></li>
+        <li><a class=\"fridge\" href=\"index.php?category=fridge\">Fridge</a></li>
+        <li><a class=\"freezer\" href=\"index.php?category=freezer\">Freezer</a></li>
+        <li><a class=\"produce\" href=\"index.php?category=produce\">Produce</a></li>
+        <li><a class=\"other\" href=\"index.php?category=other\">Other</a></li>");
+        break;
+        case 'fridge':
+            return ("<li><a class=\"all\" href=\"index.php?category=all\">All</a></li>
+        <li><a class=\"pantry\" href=\"index.php?category=pantry\">Pantry</a></li>
+        <li><a class=\"fridge current\" href=\"index.php?category=fridge\">Fridge</a></li>
+        <li><a class=\"freezer\" href=\"index.php?category=freezer\">Freezer</a></li>
+        <li><a class=\"produce\" href=\"index.php?category=produce\">Produce</a></li>
+        <li><a class=\"other\" href=\"index.php?category=other\">Other</a></li>");
+        break;
+        case 'freezer':
+            return ("<li><a class=\"all\" href=\"index.php?category=all\">All</a></li>
+        <li><a class=\"pantry\" href=\"index.php?category=pantry\">Pantry</a></li>
+        <li><a class=\"fridge\" href=\"index.php?category=fridge\">Fridge</a></li>
+        <li><a class=\"freezer current\" href=\"index.php?category=freezer\">Freezer</a></li>
+        <li><a class=\"produce\" href=\"index.php?category=produce\">Produce</a></li>
+        <li><a class=\"other\" href=\"index.php?category=other\">Other</a></li>");
+        break;
+        case 'produce':
+            return ("<li><a class=\"all\" href=\"index.php?category=all\">All</a></li>
+        <li><a class=\"pantry\" href=\"index.php?category=pantry\">Pantry</a></li>
+        <li><a class=\"fridge\" href=\"index.php?category=fridge\">Fridge</a></li>
+        <li><a class=\"freezer\" href=\"index.php?category=freezer\">Freezer</a></li>
+        <li><a class=\"produce current\" href=\"index.php?category=produce\">Produce</a></li>
+        <li><a class=\"other\" href=\"index.php?category=other\">Other</a></li>");
+        break;
+        case 'other':
+            return ("<li><a class=\"all\" href=\"index.php?category=all\">All</a></li>
+        <li><a class=\"pantry\" href=\"index.php?category=pantry\">Pantry</a></li>
+        <li><a class=\"fridge\" href=\"index.php?category=fridge\">Fridge</a></li>
+        <li><a class=\"freezer\" href=\"index.php?category=freezer\">Freezer</a></li>
+        <li><a class=\"produce\" href=\"index.php?category=produce\">Produce</a></li>
+        <li><a class=\"other current\" href=\"index.php?category=other\">Other</a></li>");
+        break;
```

Comment:
There's no default case here, there needs to be something that deals with a situation where what's passed to this function isn't one of the options

Reply:
And that needs to be one of the unit tests

Reply:
default added. Ok, I will test that in the unit test

Reply:
ok test done for both success and type error